v1.16/Dockerfile

```diff
@@ -38,6 +39,7 @@ RUN apk add --no-cache \
          google-protobuf:3.21.12 \
          nio4r:2.6.0 \
          snappy:0.3.0 \
+         rdkafka:0.16.0 \
```

Comment:
Thank you for your contribution @onecer, please add `rdkafka` to the `outputs/Gemfile` and update the `Gemfile.lock` as well.

Reply:
same as https://github.com/kube-logging/fluentd-images/pull/141#discussion_r1671790820